### ISSUE TITLE
ebay-file-preview-card: Missing A11y text for menu_button

### ISSUE THREAD

**Issue by janetwang-stack** [eBay/evo-web#390]
### I verified there's no existing issue for this bug.

- [x] There are no existing issues

### Current behavior

There is no A11y text for `menu button` of `ebay-file-preview-card`
<img width="2438" height="1136" alt="Image" src="https://github.com/user-attachments/assets/50716105-ef06-4e73-a02d-0985d803f394" />
This appears to be one of those places we don't have consistent naming of a11y text props, we should fix that too:
<img width="774" height="368" alt="Image" src="https://github.com/user-attachments/assets/886388eb-86b4-42b0-9a85-bff0a1d1fd16" />

### Expected behavior

The menu button exposes a clear accessible name (e.g., “Menu options”) so assistive technologies can announce it correctly.

### Package and version

@ebay/ebayui-core": "^15"

### Additional context

https://ebay-eng.slack.com/archives/CK8AX1L49/p1765234354167349
